static/css/styles.css

```diff
@@ -476,28 +476,44 @@ a:active {
     text-decoration: none;
 }
 
-p a {
+a {
     position: relative;
 }
 
-p a:after {
+a:after {
```

Comment:
Are you sure you want this on all existing list of the whole site (including header, nav, footer, …)?

Reply:
That's the default behavior, then I have to overwrite it if I want something for a specific element